crm_phonecall/models/res_partner.py

```diff
@@ -1,6 +1,5 @@
-# -*- coding: utf-8 -*-
-# Copyright (C) 2004-today OpenERP SA (<http://www.openerp.com>)
-# Copyright (C) 2017 Tecnativa - Vicent Cubells
+# Copyright 2004-today Odoo SA (<http://www.odoo.com>)
```

Comment:
I don't think they deserve the `-today` prefix, since it was deprecated by them long ago. Same in other files.